### ISSUE TITLE
Datasets are wrong？

### ISSUE THREAD

**Issue by lazyloafer** [kjh9503/caqr#1]
in your link, the datasets are Q2B-version, but not BetaE-version?

**Comment by kjh9503** [comment]
> in your link, the datasets are Q2B-version, but not BetaE-version?

No, it's not. Although it is separated by query type like Q2B version, it is split data of BetaE version data, not Q2B version. It was to run on the Q2B code without modification.
The data used in our experiments all originated from BetaE.
